src/Air.zig

```diff
@@ -928,21 +946,7 @@ pub fn typeOfIndex(air: Air, inst: Air.Inst.Index) Type {
         .arg,
         => return datas[inst].ty,
 
-        .assembly,
-        .block,
-        .constant,
-        .struct_field_ptr,
-        .struct_field_val,
-        .slice_elem_ptr,
-        .ptr_elem_ptr,
-        .cmpxchg_weak,
-        .cmpxchg_strong,
-        .slice,
-        .shuffle,
-        .aggregate_init,
-        .union_init,
-        .field_parent_ptr,
-        => return air.getRefType(datas[inst].ty_pl.ty),
+        .assembly, .block, .constant, .struct_field_ptr, .struct_field_val, .slice_elem_ptr, .ptr_elem_ptr, .cmpxchg_weak, .cmpxchg_strong, .slice, .shuffle, .aggregate_init, .union_init, .field_parent_ptr, .cmp_vector => return air.getRefType(datas[inst].ty_pl.ty),
```

Comment:
Whoops, sorry about that. I missed that when I paged through the diff. Will fix.

I really need to fix my editor setup so I can run `zig fmt` in real-time to catch these sooner. It really doesn't like auto formatting these huge files.

Reply:
When you say it doesn't like it, what do you mean exactly? On my machine `zig fmt` takes 0.533s to format the entire standard library and self-hosted compiler. Perhaps you are using a debug build of the compiler for `zig fmt` instead of release?

Reply:
I am indeed using a debug build, but `zig fmt` is by no means the bottleneck here. For whatever reason, Sublime Text takes a second or two to ponder the file update. VSCode handles it flawlessly, but weird Electron issues are preventing me from using it.